source/Camera/Maintenance/VerifyCameraTrueness/VerifyCameraTrueness.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+This example shows how to verify the local dimension trueness of a camera.
+If the trueness is much worse than expected, the camera may have been damaged by
+shock in shipping in handling. If so, look at the CorrectCameraTrueness sample.
+*/
+
+#include <Zivid/Zivid.h>
+
+#include <iomanip>
+#include <iostream>
+
+int main()
+{
+    try
+    {
+        Zivid::Application zivid;
+
+        std::cout << "Connecting to camera" << std::endl;
+        auto camera = zivid.connectCamera();
+
+        // For convenience, print the timestamp of the latest correction
+        if(Zivid::Experimental::Calibration::hasCameraCorrection(camera))
+        {
+            const auto timestamp = Zivid::Experimental::Calibration::cameraCorrectionTimestamp(camera);
+            const auto time = std::chrono::system_clock::to_time_t(timestamp);
+            std::cout << "Timestamp of current camera correction: " << std::put_time(std::gmtime(&time), "%FT%TZ")
+                      << std::endl;
+        }
+        else
+        {
+            std::cout << "This camera has no in-field correction written to it." << std::endl;
+        }
+
+        // Gather data
+        std::cout << "Capturing calibration object" << std::endl;
+        const auto detectionResult = Zivid::Experimental::Calibration::detectFeaturePoints(camera);
+
+        // Prepare data and check that it is appropriate for in-field verification
+        const auto input = Zivid::Experimental::Calibration::InfieldCorrectionInput{ detectionResult };
+        if(!input.valid())
+        {
+            throw std::runtime_error("Capture not valid for in-field verification! Feedback: "
+                                     + input.statusDescription());
+        }
+
+        // Show results
```

Comment:
We use present continuous tense (ing) for comments and print outs as a standard. We tend to print rather than comment, only for details we comment rather than print.

Reply:
Removing the comment.